hadoop-tools/hadoop-sls/src/main/java/org/apache/hadoop/yarn/sls/RMRunner.java

```diff
@@ -107,7 +111,11 @@ public void increaseQueueAppNum(String queue) throws YarnException {
     queueAppNumMap.put(queueName, appNum);
     SchedulerMetrics metrics = wrapper.getSchedulerMetrics();
     if (metrics != null) {
-      metrics.trackQueue(queueName);
+      try {
+        metrics.trackQueue(queueName);
+      } catch (Exception e) {
+        LOG.error("Caught exception while tracking a queue", e);
```

Comment:
can you please check if we wont throw the error from here, that wont cause any issue